main.js

```diff
@@ -1,97 +1,182 @@
-    let liMain = document.createElement("li");
-    icon.src = element.icon;
-    liTemp.innerHTML = `Temp: ${element.temp}${element.unit}`;
-    liHum.innerHTML = `Hum: ${element.humi}%`;
-    liMain.innerHTML = `${element.main}`;
-    ulElement.appendChild(icon);
-    ulElement.appendChild(liTemp);
-    ulElement.appendChild(liHum);
-    ulElement.appendChild(liMain);
-    weatherDisplay.appendChild(ulElement);
-  }
-}
+    const { temp, unit, humi, time, icon: iconSrc } = myList[i];
 
-function giveMeUnit() {
-  let radioF = document.querySelector("#F");
-  return radioF.checked ? "°F" : "°C";
-}
-function getIconURL(iconName) {
-  return ` http://openweathermap.org/img/wn/${iconName}@2x.png`;
-}
+    let mainTime = document.createElement("div");
+    mainTime.className = "main-time";
+    mainTime.innerHTML = time;
 
-function converTemp(unit) {
-  for (let i = 0; i < myList.length; i++) {
-    const element = myList[i];
-    element.unit = unit;
-    if (unit === "°F") {
-        element.temp = ((element.temp * 9) / 5 + 32).toFixed();
-    }
-    else{
-        element.temp = ((element.temp - 32) * 5/9).toFixed();
-    }
+    let mainImg = document.createElement("img");
+    mainImg.src = iconSrc;
+
+    let mainVariables = document.createElement("div");
+    mainVariables.className = "main-variables";
+    let layoutTemperature = document.createElement("div");
+    layoutTemperature.className = "layout-temperature";
+    let layoutHumidity = document.createElement("div");
+
+    let mainTemperature = document.createElement("div");
+    mainTemperature.innerHTML = temp;
+    mainTemperature.className = "main-temperature";
+
+    let mainUnit = document.createElement("div");
+    mainUnit.innerHTML = unit;
+    mainUnit.className = "main-unit";
+
+    layoutTemperature.appendChild(mainTemperature);
+    layoutTemperature.appendChild(mainUnit);
+
+    let mainHumidity = document.createElement("div");
+    mainHumidity.innerHTML = `${humi}%`;
+    mainHumidity.className = "main-humidity";
+    layoutHumidity.appendChild(mainHumidity);
+    mainVariables.appendChild(layoutTemperature);
+    mainVariables.appendChild(layoutHumidity);
+
+    let singleFullReport = document.createElement("div");
+    singleFullReport.className = "single-full-report";
+    let singleReportInfo = document.createElement("div");
+    singleReportInfo.className = "single-report";
+    singleReportInfo.appendChild(mainImg);
+    singleReportInfo.appendChild(mainVariables);
+    singleFullReport.appendChild(mainTime);
+    singleFullReport.appendChild(singleReportInfo);
+    forecastList.appendChild(singleFullReport);
   }
-}
-function handleClick(myRadio) {
+};
+
+const handleClick = (myRadio) => {
+  myRadio.checked = true;
   if (myRadio.value !== myList[0].unit) {
-    converTemp(myRadio.value);
+    console.log(myRadio.value);
+    myRadio.value === "°C"
+      ? (radioBfahrenheit.checked = false)
+      : (radioBCelsius.checked = false);

```

Comment:
Hacer asignaciones asi dentro de la condicion de un ternario no es muy legible
```suggestion
    const isCelsius = myRadio.value === "°C";
    radioBFahrenheit.checked  = !isCelsius;
    radioBCelsius.checked = isCelsius;
```